lib/unifile-fs.js

```diff
@@ -32,6 +33,19 @@ class FsConnector {
     this.name = NAME;
   }
 
+  resolvePath(path) {
```

Comment:
I'd rather use `Path.resolve(this.rootPath, path)`. Also, there is a function to check if the path is already resolved: `Path.isAbsolute(path)`